undefined_syms.txt

```diff
@@ -317,6 +319,10 @@ D_060011EC = 0x060011EC;
 D_06002260 = 0x06002260;
 D_0600238C = 0x0600238C;
 
+// z_bg_spot15_rrbox
+D_06000348 = 0x06000348;
+D_06000180 = 0x06000180;
+
```

Comment:
I'm a bit concerned about this large `undefined_syms.txt` diff. I'm not sure what github is doing because the only actual diff at the moment is indeed about en_niw (and the recently merged en_box)

Reply:
Yeah I have no idea what happened there either, I just tried to copy and paste the latest undefined_syms from the repo (probably a bad idea).